ansible/deploy-proteus-notify.yml

```diff
@@ -0,0 +1,31 @@
+- hosts: notify.proteus.test.ooni.io
+  vars:
+    proteus_notify_https_port: 443
+    letsencrypt_nginx: yes
+    letsencrypt_domains: "{{ inventory_hostname }}"
+    proteus_database_url: "postgres://proteus:{{ vault_proteus_database_password_testing }}@db-1.proteus.test.ooni.io:5432/proteus?sslmode=require"
+    proteus_auth_jwt_secret : "{{ vault_proteus_auth_jwt_secret_testing }}"
+    proteus_fcm_server_key: "{{ vault_fcm_server_key_testing }}"
+    proteus_apn_key_password: "{{ vault_proteus_apn_key_password_testing }}"
+    proteus_apn_key_content: "{{ vault_proteus_apn_key_content_testing }}"
+    proteus_environment: "production"
```

Comment:
Just checking here: it has always been `production` and everything worked for us, but I've never been sure whether this is the correct setting or not. What is the meaning of this key?